R/recall_data.R

```diff
@@ -128,10 +138,10 @@ recall_by_years <- function(start_year = 1900, end_year = 2100, limit = 25) {
     api_output <- call_vrd_api(url_, year_range, limit)
 
     # convert content to a dataframe
-    contents_df <- as.data.frame(api_output$content)
+    contents_df <- clean_vrd_api(api_output)
+
 
-    # currently just provides raw dataframe output, needs to be cleaned up
-    # any repetitive clean-up should go into helper functions
+    # if partial = FALSE need to filter data, if partial = TRUE return all data
```

Comment:
maybe this comment is in the wrong place?

Reply:
Yup fixed